tests/test_whatsapp.py

```diff
@@ -0,0 +1,37 @@
+from app.routers import whatsapp
```

Comment:
Please add an end to end test:

Use the client to fetch the `/whatsapp` route and check you get a valid JSON.